getting-started-tutorial/01-mlrun-basics.ipynb

```diff
@@ -399,24 +399,52 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Set this environment variable in case of dark site\n",
+    "# os.environ['SAMPLE_DATA_SOURCE_URL_PREFIX'] = '/v3io/bigdata/demos-data/iguazio/'"
+   ]
+  },
```

Comment:
I think we should give instructions in markdown, we don't need environment variables if the user is going to change the value within the notebook.
Also, in the text, if we give a reference to a folder, we should have it in /v3io/projects, not bigdata

Reply:
Instructions are now in markdown and folder changed to projects